### ISSUE TITLE
There is an error in the map part when learning. If you don't give A, it doesn't sit and it goes well.

### ISSUE THREAD

**Issue by codingyoutub** [AlexeyAB/darknet#8779]
There is an error in the map part when learning. If you don't give A, it doesn't sit and it goes well.

darknet.exe detector train data/obj.data cfg/yolov4_tiny.cfg yolov4-tiny.conv.29 -map 

 2519: 0.103649, 0.349622 avg loss, 0.002610 rate, 0.032000 seconds, 5038 images, 0.015937 hours left
Loaded: 0.000000 seconds
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 30 Avg (IOU: 0.797186), count: 1, class_loss = 0.615162, iou_loss = 0.054751, total_loss = 0.669913
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 37 Avg (IOU: 0.000000), count: 1, class_loss = 0.000001, iou_loss = 0.000000, total_loss = 0.000001
 total_bbox = 5256, rewritten_bbox = 0.000000 %
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 30 Avg (IOU: 0.744713), count: 1, class_loss = 0.877250, iou_loss = 0.034362, total_loss = 0.911612
v3 (iou loss, Normalizer: (iou: 0.07, obj: 1.00, cls: 1.00) Region 37 Avg (IOU: 0.000000), count: 1, class_loss = 0.000002, iou_loss = 0.000000, total_loss = 0.000002
 total_bbox = 5257, rewritten_bbox = 0.000000 %

 (next mAP calculation at 2520 iterations)
 Last accuracy mAP@0.50 = 49.79 %, best = 49.79 %
 Tensor Cores are disabled until the first 3000 iterations are reached.

 2520: 0.381120, 0.352771 avg loss, 0.002610 rate, 0.032000 seconds, 5040 images, 0.015928 hours left

 calculation mAP (mean average precision)...
 Detection layer: 30 - type = 28
 Detection layer: 37 - type = 28
92

**Comment by stephanecharette** [comment]
See issue #8669.
